contrib/trace_propagators/build.gradle

```diff
@@ -12,6 +12,8 @@ ext.moduleName = "io.opentelemetry.contrib.trace.propagation"
 dependencies {
     api project(':opentelemetry-api')
 
+    implementation libraries.guava
```

Comment:
We add guava as dependency and this was raised as an issue to us. Can we just copy the padding function for the moment.

Reply:
I looked that prometheus exporter and several contribs to sdk already use guava and thought it safe to use it. I would be very happy to remove guava library from Otel. But probably not in this PR?

Reply:
The issue is that this is an API contrib package, rather than SDK, so we don't want to introduce the guava dependency where it wasn't before.

Reply:
Got it. Will copy padding function from Guava then.

Reply:
That's correct. We don't really need to obstruct the usage of Guava (and similar libraries), but we want to decrease whenever possible.

Reply:
Thanks @iNikem